clean.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+
+rm -rf /tmp/s3
+rm -rf /tmp/s4
+
+mkdir /tmp/s3
+mkdir /tmp/s4
```

Comment:
can this script be removed, and its remove-and-recreate actions be done in the golang test which creates/uses them ?